src/patient-registration/patient-registration.component.tsx

```diff
@@ -178,8 +178,7 @@ export const PatientRegistration: React.FC = () => {
         let defaultValue = getValueIfItExists(name, 'elementDefaults', templateXmlDoc);
         initialFormValues[name] = defaultValue ?? '';
       });
-
-      setAddressValidationSchema(addressValidationSchemaTmp);
```

Comment:
Can you also delete the declaration of this state object? Since we're no longer maintaining `addressValidationSchema` as a state, we need the variable.

Reply:
Fixed.